style.css

```diff
@@ -579,6 +1183,99 @@ body {
     margin-top: 116px;
     transform: rotate(135deg);
   }
+
+  .order {
+    flex-direction: column;
+    padding-bottom: 25px;
+  }
+
+  .order-image__first,
+  .order-image__second,
+  .order-number,
+  .range-1 {
+    display: none;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)